### ISSUE TITLE
Add term crescent shape iliac bone

### ISSUE THREAD

**Issue by lisannesw** [obophenotype/human-phenotype-ontology#10416]
**Preferred term label:**
Crescent shape iliac bone

**Synonyms**
paraglider shape iliac bone, crescent-shaped iliac wing

**Definition (free text, please give PubMed ID)**
Deficient mineralization of the iliac wings, giving a paraglider/crescent shape to the iliac bone
![image](https://github.com/obophenotype/human-phenotype-ontology/assets/160679899/3d20c104-2672-4e6e-9467-bfed71435948)
PMID: 28744080
PMCID: PMC6611735

**Parent term (use hpo.jax.org/app)**
Abnormal ilium morphology HP:0002867

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Achondrogenesis (omim200600, 200610) PMID: 28744080
Metatropic dysplasia (omim156530)

**Your nano-attribution (ORCID)**
Andreas Zankl: 0000-0001-8612-1062, @azankl
Lisanne Wit: 0009-0007-1020-5412

**Comment by pnrobinson** [comment]
Adding new term: 
- ID: HP:6000653
- label: Crescent-shaped iliac bone
- parents: [Abnormal rib morphology (HP:0000772)](https://hpo.jax.org/app/browse/term/HP:0000772)
- pmid: PMID:28744080
- def: Deficient mineralization of the iliac wings, giving a paraglider/crescent shape to the iliac bone.
- comment: 
- orcid: orcid:0000-0001-8612-1062
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10416
- synonyms: Crescent-shaped iliac wing(NONE)|Paraglider shape iliac bone(NONE)
